x/incentive/params.go

```diff
@@ -9,7 +9,7 @@ import (
 const secondsPerDay = 86400
 
 var (
-	defaultMaxUnbondings           = uint32(20)
+	defaultMaxUnbondings           = uint32(5)
```

Comment:
This is not per-denom-and-tier, rather than account-wide. Allows smaller default value, and also scales better with large number of listed assets.